docs/content/docs/features/table-types.md

```diff
@@ -102,6 +102,9 @@ Current supported aggregate functions and data types are:
 * `listagg`: supports STRING data type.
 * `bool_and` / `bool_or`: support BOOLEAN data type.
 
+Most aggregate functions do not support retraction (`UPDATE_BEFORE` and `DELETE`). If you allow some functions to ignore retraction messages, you can configure:
```

Comment:
Could this explanation add into < hint info >？

Reply:
I think this is normal documentation, no need to be hint.